run.cmd

```diff
@@ -0,0 +1,40 @@
+@if not defined _echo @echo off
+setlocal
+
+if not defined VisualStudioVersion (
+  if defined VS140COMNTOOLS (
+    call "%VS140COMNTOOLS%\VsDevCmd.bat"
+    goto :Run
+  )
+  echo Error: Visual Studio 2015 required.
+  echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
```

Comment:
BTW: These instruction point to corefx. corert build instructions and prerequisites are different.